src/_locales/en/messages.json

```
@@ -204,6 +204,18 @@
 	"home": {
 		"message": "Home"
 	},
+	"initialSyncTitle": {
+		"message": "Sync full history?"
+	},
+	"initialSyncDescription": {
+		"message": "This is the first time that you are accessing this service. Would you like to sync the full history right now? This may take a while depending on how big your history is.\n\nBefore choosing \"Yes\", make sure that the options below are adjusted to your liking.\n\nIf you choose \"No\", you will not be asked this again, unless you clear the last sync date for the service or clear the storage."
```

```suggestion
		"message": "This is the first time that you are accessing this service. Would you like to sync the full history right now? This may take a while depending on how big your history is.\n\nBefore choosing \"Yes\", make sure that the options below are adjusted to your liking.\n\nIf you choose \"No\", you will not be asked this again, unless you clear the last sync date for the service or clear the storage. You can always trigger the sync manually."
```
Makes it less of a now-or-never condition.